src/kernel/chainparams.cpp

```diff
@@ -250,7 +250,7 @@ class CTestNetParams : public CChainParams {
         pchMessageStart[3] = 0x07;
         nDefaultPort = 18333;
         nPruneAfterHeight = 1000;
-        m_assumed_blockchain_size = 93;
+        m_assumed_blockchain_size = 181;
```

Comment:
In 13f2815f4caca68a3ab0d0b865f370956308cad6 "[kernel] update assumed blockchain and chainstate sizes for v29"

This seems excessively high, I'd be surprised if testnet3's datadir size doubled in the past 6 months.

Across my 3 nodes, I see 111 GiB, 112 GiB, and 116 GiB, so I'd expect this to be 127.

Reply:
It is a big jump, but `du .bitcoin/testnet3 -h` gives me 181G. Let's see what other people have?

Reply:
I have atleast 154GB, and assume it's going to end up around the 180GB mark. Given that, it seems this should be something closer to 200?

Reply:
> It is a big jump, but `du .bitcoin/testnet3 -h` gives me 181G. Let's see what other people have?

In the past, I've done `du -csh chainstate/ blocks/` so that indexes, debug.log, and wallets, are not included in the size.

```
du -csh ~/.bitcoin/testnet3/chainstate/ ~/.bitcoin/testnet3/blocks/
11G	/home/ava/.bitcoin/testnet3/chainstate/
101G	/home/ava/.bitcoin/testnet3/blocks/
111G	total
```

Reply:
```bash
du -csh chainstate/ blocks/
 13G	chainstate/
167G	blocks/
180G	total
```

Reply:
du -csh ~/.bitcoin/testnet3/blocks/ ~/.bitcoin/testnet3/chainstate/
167G    /home/chris/.bitcoin/testnet3/blocks/
13G     /home/chris/.bitcoin/testnet3/chainstate/
179G    total

Reply:
bumped to 200, roughly 10% more than 180

Reply:
```
168G	/home/martin/.bitcoin/testnet3/blocks/
13G	/home/martin/.bitcoin/testnet3/chainstate/
181G	total
```
with a node that is offline 99% of the time and therefore did not experience many of the reorg shenanigans.

Reply:
Got 180GiB across blocks and chainstate for testnet3 too.